extensions/wikia/ArticleVideoMVP/templates/ArticleVideoMVP_index.php

```diff
@@ -0,0 +1,17 @@
+<div id="premium-mvp-video" style="background-image: url(<?= $thumbnailUrl; ?>);">
```

Comment:
having images as a background image isn't very SEO friendly. Did you consider using `img` tag instead?